### DISCUSSION TITLE
How "Required Score to Rank Up" works ?

### DISCUSSION THREAD

**Discussion by M0hanrajp** [codewars/codewars.com#3309] [Q&A] [answered]
From the document [Required Score to Rank Up](https://docs.codewars.com/gamification/ranks/)

I'm not able to understand, we enter in Codewars with 0 honor, at rank 8 kyu.
I need to score 20 kyu in order to progress and reach 7 kyu.

- is score & honor the same ?
- do we rank up only if we choose "Rank up" option and solve challenges ?

Thanks.

![image](https://github.com/user-attachments/assets/4edc35d2-693c-4e71-b082-91982615f892)

**Comment by M0hanrajp** [comment]
I'm being assigned only 2KYU for completing 7KYU Kata. Please let me know if there is any update.
Thanks

**Reply by BobtheLantern** [reply]
Honor and rank points are different. Absolute rank points are not displayed on your profile, but honor is. Just to clarify "Kyu" is a rank title, it is not synonymous with "points". I believe what you mean to say is that you are being awarded 2 honor for completing a 7-kyu kata, and that would be correct. The kata with white icons (7 and 8 kyu kata) both award 2 honor, but 8 kyu kata award 2 rank points and 7 kyu kata award 3 rank points. Again, absolute rank points are not displayed on your profile, but you can see them by using the api. At the time of this post, you have 516 honor and 380 rank points.

**Reply by M0hanrajp** [reply]
Hi, Thanks for your response,
could you please help to clarify the following ?

>but 8 kyu kata award 2 rank points and 7 kyu kata award 3 rank points.

These "Rank points" are considered for progressing XKYU ranks right ? based on the table in Comment 1, ? like I would need 20 rank points to rank 7KYU from rank 8KYU.

And only these rank points are considered for ranking up from 8KYU to 1 dan ?

Thanks.

**Reply by BobtheLantern** [reply, marked as answer]
Yes, rank points are what determine your overall rank. Once you reach 20 rank points, your rank will be 7kyu. You are currently at 380 rank points which means you are currently ranked 5 kyu. Once you reach 643 rank points, you will advance to 4 kyu. Only these points matter for your rank, honor doesn't matter at all for your rank. Honor is more of a measure of community activity and the more honor you have, the more privileges you have on codewars (things like being able to vote on a beta kata's rank and unmarking comments as spoilers are examples of privileges, if you want to know more about honor you can read the docs on codewars). For rank points, the higher the difficulty of kata that you complete, the more rank points you get, and the most difficult katas earn quite a lot of rank points. An 8 kyu kata only awards 2 rank points, but a 1 kyu kata awards 1097.

**Reply by M0hanrajp** [reply]
Thanks for your response.
